### ISSUE TITLE
Windows Titlebar and Menubar

### ISSUE THREAD

**Issue by luckydye** [anyproto/anytype-ts#408]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing issues and didn't find any that were similar
- [X] I have considered creating a pull request with fixes instead of a bug report and want to proceed

### Current Behavior

The Titlebar and Menubar on Windows look different from the Mac Version.

Windows
![image](https://github.com/anyproto/anytype-ts/assets/10381895/3a2ffd26-09a0-4256-a49d-62f7cbd5fec0)

Mac
![image](https://github.com/anyproto/anytype-ts/assets/10381895/37adea8d-474d-4ed1-8e97-86ff8d84928d)

### Expected Behavior

I think the Titlebar and Menubar should look the same, or at least similar, on both platforms. Plus the Windows menubar is just pretty ugly. In other apps on windows, the menubar is hidden inside a popover menu for example.

Im thinking it could work like this:
![image](https://github.com/anyproto/anytype-ts/assets/10381895/5b1c8882-b68e-4f9d-9d28-968eee9ec6db)

### Steps To Reproduce

1. Install anytype desktop
2. Open anytype desktop

### Environment

```markdown
- OS: Windows
- Version: Windows 11 23H2
```

### Anything else?

Im willing to contribute for this!

**Comment by zhouxinghong** [comment]
I think so too👍 +1

**Comment by zhouxinghong** [comment]
Should use a modern hamburger menu style

**Comment by ra3orblade** [comment]
This is definitely not a bug.

**Comment by ruizlenato** [comment]
Any news?

**Comment by ruizlenato** [comment]
> Any news?

**My concept**
<img width="1169" height="761" alt="Image" src="https://github.com/user-attachments/assets/8648ba15-88f4-4623-9e68-1e4c389bd400" />

**Comment by ranpy13** [comment]
Yup, it is very frustrating. Same story as on dayone app on windows. It feels like it was made for mac first, and then lazily ported over to windows using a web wrapper. This however seems a little different, I will be taking this up and trying to fix it.

**Comment by ra3orblade** [comment]
Windows menu bar has been updated, closing as completed.

**Comment by ruizlenato** [comment]
> Windows menu bar has been updated, closing as completed.

### **It looks so bad!**
It's more like a nonsense bar. They could have integrated the buttons into the bar like they do in macOS.
<img width="1160" height="231" alt="Image" src="https://github.com/user-attachments/assets/cd9a740a-a4c5-45fc-84cf-6412c588207f" />


Like this (my concept): 
<img width="1169" height="148" alt="Image" src="https://github.com/user-attachments/assets/5a9c687d-499f-4c7e-939e-5c3be5f54260" />


They paid much more attention to the macOS version and forgot to pay attention to the Windows and Linux versions. How sad.
<img width="1919" height="191" alt="Image" src="https://github.com/user-attachments/assets/ff0c3488-6e6c-4488-8d2d-8366aa88c0df" />
